deepchecks/utils/metrics.py

```diff
@@ -89,6 +89,75 @@ class ModelType(enum.Enum):
 }
 
 
+class DeepcheckScorer:
+    """Encapsulate scorer function with extra methods.
```

Comment:
Explain here what a scorer is, refer to https://scikit-learn.org/stable/modules/model_evaluation.html

Reply:
Why? it's internal class

Reply:
Still people may wanna use it, but can solve in future PR